JS/Airport.js

```diff
@@ -1,120 +1,99 @@
     }
 
-    constructor(planes) {
-        this.planes = planes;
+    getTransportMilitaryPlanes() {
+        return this.getMilitaryPlanesWithType(militaryTypes.TRANSPORT);
     }
 
-
-    getExperimentalPlanes() {
-        let experimentalPlanes  = [];
-        this.planes.forEach(plane => {
-            if (plane instanceof experimentalPlane) {//if
-                experimentalPlanes.push(plane);
-            }
-        });
-//return
-        return experimentalPlanes;
+    getBomberMilitaryPlanes() {
+        return this.getMilitaryPlanesWithType(militaryTypes.BOMBER);
     }
 
-
-
     sortByMaxDistance() {
-        this.planes.sort((a, b) => (a.Get_Max_Flight_Distance() > b.Get_Max_Flight_Distance()) ? 1 : -1);
+        this.planes.sort((currentPlane, nextPlane) => (currentPlane.getMaxFlightDistance() > nextPlane.getMaxFlightDistance()) ? 1 : -1);
         return this;
     }
 
-    /**
-     * Sorts by max speed
-     * @return Airport
-     */
     sortByMaxSpeed() {
-        this.planes.sort((a, b) => (a.getMS() > b.getMS()) ? 1 : -1);
+        this.planes.sort((currentPlane, nextPlane) => (currentPlane.getMaxSpeed() > nextPlane.getMaxSpeed()) ? 1 : -1);
         return this;
     }
 
     sortByMaxLoadCapacity() {
-        this.planes.sort((a, b) => (a.getMinLoadCapacity() > b.getMinLoadCapacity()) ? 1 : -1);
+        this.planes.sort((currentPlane, nextPlane) => (currentPlane.getMaxLoadCapacity() > nextPlane.getMaxLoadCapacity()) ? 1 : -1);
         return this;
     }
 
-    getPlanes() {
-        return this.planes;
+    getMilitaryPlanesWithType(militaryPlaneType) {
+        let militaryPlanesWithType = [];
+        let militaryPlanes = this.getMilitaryPlanes();
+        for (let i = 0; i < militaryPlanes.length; i++) {
+            if (militaryPlanes[i].getMilitaryType() === militaryPlaneType) {
+                militaryPlanesWithType.push(militaryPlanes[i]);
+            }
+        }
+        return militaryPlanesWithType;
     }
 
+    getExperimentalPlanesWithClassificationLevel(classificationLevel) {
+        let experimentalPlanesWithClassificationLevel = [];
+        let experimentalPlanes = this.getExperimentalPlanes();
+        for (let i = 0; i < experimentalPlanes.length; i++) {
+            if (experimentalPlanes[i].classificationLevel === classificationLevel) {
+                experimentalPlanesWithClassificationLevel.push(experimentalPlanes[i]);
+            }
+        }
+        return experimentalPlanesWithClassificationLevel;
+    }
 
+    getSpecificPlanes(planeType) {
+        let specificPlanes = [];
+        this.planes.forEach(plane => {
+            if (plane instanceof planeType) {
+                specificPlanes.push(plane);
+            }
+        });
+        return specificPlanes;
+    }
 
+    getPlanes() {
+        return this.planes;
+    }
 
     static print(planes) {
         return JSON.stringify(planes);
     }
 }
 
-module.exports = Airport;
+module.exports = Airport;
```

Comment:
The best practice to use the following syntax while exporting variables from the file:
`module.exports = {
Airport
}`